git-extra/getgit

```diff
@@ -274,4 +284,4 @@ fi
 echo "Git is now available! If you want to add \"Git GUI Here\" and \"Git Bash"
```

Comment:
These messsges should probably be modified if the context menu scripts aren't created.